code/index.html

```diff
@@ -5,32 +5,64 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <link rel="stylesheet" href="style.css" />
+    <link rel="icon" href="./assets/questionmark.jpg"  type="image/x-icon" />
     <title>Technigo Project Cluedo</title>
   </head>
   <body>
+
+<!-- HEADER -->
+
+    <header>
+        <h1>CLUEDO - who did it?</h1>
+    </header>
+
+<!-- SECTION CARDS-->
+
     <section class="cards">
-      <article class="killer-deck" id="killerCard">
-        <p>KILLER</p>
-        <p id="killerName"></p>
-        <p id="killerAge"></p>
-        <p id="killerOccupation"></p>
-        <img id="killerImage" src="" height="100" />
-        <p id="killerDescription"></p>
-      </article>
-      <article class="weapon-deck" id="weaponCard">
-        <p>WEAPON</p>
-        <p id="weaponName"></p>
-        <p id="weaponWeight"></p>
+
+        <article class="killer-deck allCards" id="killerCard">
```

Comment:
Maybe here the indentation could be one line left?

Reply:
Indeed, should be two spaces.  This looks like tabs